csc_335_restaurant/src/model/Sales.java

```diff
@@ -1,5 +1,99 @@
+/*
+ * The Sales class keeps track of how many times an item has been ordered using
+ * a HashMap.
+ * */
 package model;
 
-public class Sales {
+import java.util.*;
+
 
+public class Sales {
+	/* The sales HashMap keeps track of how many of each item is sold
+	 * The itemRevenue HashMap keeps track of how many of each item is sold */
+	private HashMap<Food, Integer> sales;
+	private HashMap<Food, Double> itemRevenue;
+	
+	// Constructor method
+	public Sales() {
+		sales = new HashMap<Food, Integer>();
+		itemRevenue = new HashMap<Food, Double>();
+	}
+	
+	
+	/* This method adds all the food in bill to the hashmap by checking the food name
+	 * and adding the quantity
+	 *  */
+	public void addCompletedOrder(Bill bill) {
+		for (FoodData food: bill.getOrder()) {
+			Food copyFood = new Food(food.getName(), food.getType(), food.getPrice());
```

Comment:
This is a much simpler method that don't require to change FoodData
```suggestion
			Food copyFood = (Food) food;
```